src/client/instant-meilisearch-client.ts

```diff
@@ -1,19 +1,20 @@
-import { MeiliSearch } from 'meilisearch'
 import {
-  InstantMeiliSearchOptions,
-  InstantMeiliSearchInstance,
-  AlgoliaSearchResponse,
   AlgoliaMultipleQueriesQuery,
+  AlgoliaSearchResponse,
+  InstantMeiliSearchInstance,
+  InstantMeiliSearchOptions,
   SearchContext,
 } from '../types'
+import { SearchCache, cacheFirstFacetDistribution } from '../cache/'
 import {
-  adaptSearchResponse,
-  adaptSearchParams,
   SearchResolver,
+  adaptSearchParams,
+  adaptSearchResponse,
 } from '../adapter'
-import { createSearchContext } from '../contexts'
-import { SearchCache, cacheFirstFacetDistribution } from '../cache/'
+
+import { MeiliSearch } from 'meilisearch'
 import { constructClientAgents } from './agents'
+import { createSearchContext } from '../contexts'
```

Comment:
I don't the order of import needs to be changed, could you rollback?

Reply:
The import order has been reverted.